controller/workitem.go

```diff
@@ -671,11 +693,15 @@ func (c *WorkitemController) ListChildren(ctx *app.ListChildrenWorkitemContext)
 	return ctx.ConditionalEntities(result, c.config.GetCacheControlWorkItems, func() error {
 		var response app.WorkItemList
 		application.Transactional(c.db, func(appl application.Application) error {
+			wits, err := loadWorkItemTypesFromArr(ctx.Context, appl, result)
```

Comment:
Looks like this call and the below call to get children are under `ctx.ConditionalRequest`. I think these calls can be moved under the above transaction wrapper.

Reply:
Done in a154a7af